.eslintrc.json

```diff
@@ -10,7 +10,7 @@
         ],
         "linebreak-style": [
             "error",
-            "unix"
+            "windows"
```

Comment:
This needs to stay "unix". Is your editor not respecting the linebreak style that's already in the files?